docs/services/sda.md

```diff
@@ -10,12 +10,12 @@ It can be used as part of a [Federated EGA](https://ega-archive.org/federated) o
 
 The SDA submission pipeline has four main steps:
 
-1. [Ingest](cmd/ingest.md) splits file headers from files, moving the header to the database and the file content to the archive storage.
-2. [Verify](cmd/verify.md) verifies that the header is encrypted with the correct key, and that the checksums match the user-provided checksums.
-3. [Finalize](cmd/finalize.md) associates a stable accessionID with each archive file and backups the file.
-4. [Mapper](cmd/mapper.md) maps file accessionIDs to a datasetID.
+1. [Ingest](ingest/) splits file headers from files, moving the header to the database and the file content to the archive storage.
```

Comment:
this needs to be fixed in `sensitive-data-archive`

Reply:
I don't get it? The file `aggregate-mappings.json` sets up that for example the `cmd/mapper.md` should be generated in `services/mapper.md` or am I misunderstanding you?

Reply:
Ah, then the `aggregate-mappings.json` is what should have been updated to place the documents in the correct place.

Reply:
that and maybe this https://github.com/neicnordic/sensitive-data-archive/blob/main/sda/sda.md (i am updating this)